### PR TITLE
added ZX-Spectrum section and ZX-Poly emulator link

### PR DESCRIPTION
**Pull request by raydac**
I have added ZXSpectrum section by analogue with previously listed sections and added link to [ZXPoly](https://github.com/raydac/zxpoly) emulator

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,8 @@ Please read [CONTRIBUTING](/CONTRIBUTING.md).
   - [PS4](#ps4)
   - [PSP](#psp)
   - [PSVita](#psvita)
-
+- [ZXSpectrum](#zxspectrum)
```
Comment (alnacle) [inline README.md]: I think "ZX Spectrum" looks better.

File: README.md [hunk 2]
```diff
@@ -36,7 +36,8 @@ Please read [CONTRIBUTING](/CONTRIBUTING.md).
...
+  - [ZXPoly](#zxpoly)
```
Comment (alnacle) [inline README.md]: This is not needed.

**Review by alnacle — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please check my comments.

File: README.md (hunk 1 above)
Comment (raydac) [reply README.md]: ok

**Comment by raydac** [comment]
@alnacle it looks like that I have fixed all requests, or I have skipped something?

**Review by alnacle — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**